build/commands/lib/config.js

```diff
@@ -618,6 +618,9 @@ Config.prototype.buildArgs = function () {
 
     args.android_aab_to_apk = this.androidAabToApk
 
+    // This optimization causes crash on Android 8 and Android 8.1 arm64 devices.
+    args.use_relr_relocations = false
```

Comment:
do we have an issue to investigate the underlying problem here?

Reply:
why is this being applied to android 8/8.1 anyway? `is_high_end_android && is_official_build`

Reply:
oic `is_high_end_android = target_cpu == "arm64"`

Reply:
Created a follow up issue https://github.com/brave/brave-browser/issues/36176

Reply:
@bridiver 
underlying problem is according to [comments](https://source.chromium.org/chromium/chromium/src/+/main:chrome/android/chrome_common_shared_library.gni;l=18-25;drc=f4a00cc248dd2dc8ec8759fb51620d47b5114090?q=chrome%2Fandroid%2Fchrome_common_shared_library.gni&ss=chromium%2Fchromium%2Fsrc)
```
  # Compile libmonochrome.so with RELR relocations, which are smaller and more
  # efficient, but supported only on API 28+.
  # This is not permanently enabled for Trichrome because it shares its native
  # library with Monochrome.
```
`use_relr_relocations` is not supposed to work on Android 8/8.1 . But we install the same lib on any arm64 device.